Form/EventListener/ResizeFormListener.php

```diff
@@ -63,8 +63,8 @@ public static function getSubscribedEvents()
     {
         return array(
             FormEvents::PRE_SET_DATA    => 'preSetData',
-            FormEvents::PRE_BIND        => 'preBind',
-            FormEvents::BIND            => 'onBind',
+            FormEvents::PRE_SUBMIT      => 'preSubmit',
```

Comment:
> just add a note

Agree.

> raise an exception if thoses methods are called.

Well, if people doesn't override subscribed events, old method will be **never** called.